vendor/cegui-0.4.0-custom/src/CEGUIMouseCursor.cpp

```diff
@@ -149,12 +149,18 @@ void MouseCursor::offsetPosition(const Point& offset)
 /*************************************************************************
     Set the mouse cursor's alpha
 *************************************************************************/
-void MouseCursor::setAlpha(float alpha)
+void MouseCursor::setAlpha(float r, float g, float b, float alpha) noexcept
 {
-    d_colourRect.setColours(colour(1.0f, 1.0f, 1.0f, alpha));
+    d_colourRect.setColours(colour(r/255.f, g/255.f, b/255.f, alpha));
 }
 
-
+/*************************************************************************
+    Set the mouse cursor's color
+*************************************************************************/
+void MouseCursor::setColor(float r, float g, float b, float alpha) noexcept
+{
+    d_colourRect.setColours(colour(r/255.f, g/255.f, b/255.f, alpha/255.f));
+}
```

Comment:
If the setColours function is not noexcept, then setColour should also not be noexcept